### PR TITLE
Style Changes

### PR DESCRIPTION
**Pull request by vipineth**
[x] Fix message typo
[x] Fix button style (Referral page)
[x] Fix PositionList order error
[x] Increase `min-width` of tooltip to `28rem`
[x] Add `Modal-scrollable` to modals by default

---
- To see the specific tasks where the Asana app for GitHub is being used, see below:
  - https://app.asana.com/0/0/1203443770657415
  - https://app.asana.com/0/0/1203136653435862
  - https://app.asana.com/0/0/1203247464865883
  - https://app.asana.com/0/0/1203222378802918
  - https://app.asana.com/0/0/1203061505813273

### CODE REVIEW

File: src/components/Exchange/OrdersList.js [hunk 1]
```diff
@@ -229,11 +229,23 @@ export default function OrdersList(props) {
 
       const error = getOrderError(account, order, positionsMap);
       const orderId = `${order.type}-${order.index}`;
+      const orderTitle = `${order.type === INCREASE ? t`Increase` : t`Decrease`} ${indexTokenSymbol} ${
```
Comment (divhead) [inline src/components/Exchange/OrdersList.js:232]: Not critial
Now this place is a bit difficult to read for me, maybe put the conditions in separate variables?

Like this
```
const orderTypeText = order.type === INCREASE ? t`Increase` : t`Decrease`
const longShortText = order.isLong ? t`Long` : t`Short`;

const orderTitle = `${orderTypeText} ${indexTokenSymbol} ${longShortText}  by ${formatAmount(order.sizeDelta, USD_DECIMALS, 2, true)}`

```

File: src/components/Exchange/OrdersList.js [hunk 2]
```diff
@@ -229,11 +229,23 @@ export default function OrdersList(props) {
...
+        order.isLong ? t`Long` : t`Short`
+      }
+      by ${formatAmount(order.sizeDelta, USD_DECIMALS, 2, true)}`;
       const orderText = (
         <>
-          {order.type === INCREASE ? t`Increase` : t`Decrease`} {indexTokenSymbol} {order.isLong ? t`Long` : t`Short`}
-          &nbsp;by ${formatAmount(order.sizeDelta, USD_DECIMALS, 2, true)}
-          {error && <div className="Exchange-list-item-error">{error}</div>}
```
Comment (divhead) [inline src/components/Exchange/OrdersList.js:236]: &nbsp; was missed from the previous version

Comment (divhead) [reply src/components/Exchange/OrdersList.js:236]: Or is that okay here?)

File: src/components/Exchange/OrdersList.js [hunk 3]
```diff
@@ -397,13 +409,23 @@ export default function OrdersList(props) {
       const collateralUSD = getUsd(order.purchaseTokenAmount, order.purchaseToken, true, infoTokens);
 
       const error = getOrderError(account, order, positionsMap);
-
+      const orderTitle = `${order.type === INCREASE ? t`Increase` : t`Decrease`} ${indexTokenSymbol} ${
+        order.isLong ? t`Long` : t`Short`
```
Comment (divhead) [inline src/components/Exchange/OrdersList.js:413]: The same code is repeated three times, maybe it makes sense to create a function for this?

**Review by gdev8317 — APPROVED** [review]

**Pull request merged**